tyk-single-dc/templates/NOTES.txt

```diff
@@ -0,0 +1,24 @@
+Thank you for installing Tyk Data Plane Chart {{ .Chart.Name }}. Your release is named {{ .Release.Name }}. At this point, Tyk Data Plane is fully installed and should be accessible.
+
+{{ $gwDependency := "tyk-gateway" }}
+{{- range $i, $dep := .Chart.Dependencies }}
+{{ if contains "gateway" ($dep.Name | quote) }}
+{{ $gwDependency = $dep.Name }}
+{{ end }}
+{{- end }}
+
+{{- if index .Values "tyk-gateway" "gateway" "control" "enabled" }}
+To quickly test everything is ok, you can port-forward gateway pod to 9696 on your machine:
+    kubectl port-forward service/gateway-control-svc-{{ include "tyk-gateway.fullname" . }}-{{ $gwDependency }} 9696:{{ index .Values "tyk-gateway" "gateway" "control" "port" }}
+
+and run `curl localhost:9696/hello` to see some data back that includes redis and rpc info.
+{{- else }}
+To quickly test everything is ok, you can port-forward gateway pod to 8080 on your machine:
+    kubectl port-forward service/gateway-svc-{{ include "tyk-gateway.fullname" . }}-{{ $gwDependency }} 8080:{{ index .Values "tyk-gateway" "gateway" "service" "port" }}
+
+and run `curl localhost:8080/hello` to see some data back that includes redis and rpc info.
+{{- end }}
+
+Manage Ingress resources and APIs
+You might want to install Tyk Operator next to manage Ingress resources or manage your APIs.
+[Tyk Operator](https://github.com/TykTechnologies/tyk-operator/)
```

Comment:
this is only relevant for gateway or data plane installation..

Reply:
removed outdated info